topsort-api-v2.yml

```diff
@@ -183,11 +182,14 @@ paths:
       summary: Report events
       operationId: reportEvents
       requestBody:
-        description: >
+        description: | # Pipe preserves newlines
           Use the `/events` endpoint to notify Topsort about significant consumer interactions on the marketplace app:
-          impressions — ad promotions become visible to the consumer;
-          clicks — the consumer clicks on an ad promotion;
-          and purchases — the consumer buys some products.
+          - **Impressions** — an impression means a promotable has become visible to the consumer.
```

Comment:
I'm not very sure of this description, because it can lead the marketplaces to send impressions one by one :(